fetching/strat.py

```diff
@@ -42,12 +45,7 @@ def _is_test_file(self, path: pathlib.Path) -> bool:
 
 
 class PyTestStrategy(ApplicationStrategy):
-    FUNCTION_PATTERN = re.compile(r"[\w\s]*test_[\w\s]*\([\w\s]*\)[\w\s]*:[\w\s]*")
-    SYS_IMPORT = "import sys\n"
-    PYTYPE_IMPORTS = "from tracing import register, entrypoint\n"
-    REGISTER = "@register()\n"
-    ENTRYPOINT = "\n@entrypoint()\ndef main():\n  ...\n"
-
+    FUNCTION_PATTERN = re.compile(r"test_")
```

Comment:
This regex can be removed in favour of using `name.startswith("test_")`. Otherwise it all looks fine

Reply:
The transformer should be working for any type of strategy. Using regex pattern matching is more generalized than checking for startswith.

Reply:
Ok I can live with that